app/scripts/move-popup/dimTalentGrid.directive.js

```diff
@@ -81,5 +81,20 @@
       vm.numColumns = _.max(vm.talentGrid.nodes, 'column').column + 1 - vm.hiddenColumns;
       vm.numRows = vm.perksOnly ? 2 : (_.max(vm.talentGrid.nodes, 'row').row + 1);
     }
+
+    vm.onScopeClick = function(node) {
+      // Add a better check if is a scope node
+      if (node.column === 1) {
```

Comment:
There may be something in the item hash we're not using that identifies it as a scope